src/Common/tests/System/PlatformDetection.cs

```diff
@@ -23,6 +23,8 @@ public static class PlatformDetection
             File.Exists(Path.Combine(Environment.GetEnvironmentVariable("windir"), "regedit.exe")));
         public static bool IsWindows10Version1607OrGreater { get; } = IsWindows &&
             GetWindowsVersion() == 10 && GetWindowsMinorVersion() == 0 && GetWindowsBuildNumber() >= 14393;
+        public static bool IsWindows10VersionInsiderPreviewOrGreater { get; } = IsWindows &&
```

Comment:
"IsWindows10VersionInsiderPreviewOrGreater" This should get renamed to something more stable.  Every build of Windows 10 that isn't an RTM version is an "Insider Preview".

It would be more accurate to at least call this "IsWindows10CreatorsUpdatePreviewOrGreater".